types/merge-images/index.d.ts

```diff
@@ -14,7 +17,9 @@ declare function mergeImages(
 ): Promise<string>;
 
 declare namespace mergeImages {
-    type ImageSource = string | { src: string; x?: number; y?: number; opacity?: number };
+    type Image = string | Buffer;
```

Comment:
I don't get it. You can set a `Buffer` as a value of `img.src`? What `Image` implementation is it? Does it work in a browser?

Reply:
> You can set a `Buffer` as a value of `img.src`. 


The docs say that `img.src` can be "Array of valid image sources for new Image()."

> "Does it work in a browser?"

No, `Buffer` is a nodejs type

Reply:
> "What `Image` implementation is it?"

@BendingBender this is for nodejs, where you have to use the `Image` implementation from [node-canvas](https://npm.im/canvas). 

node-canvas's `Image` accepts `string | Buffer` - [see here](https://github.com/Automattic/node-canvas/blob/master/types/index.d.ts#L232-L236)

Reply:
I think it's OK then.